synapse/models/inet.py

```diff
@@ -3306,6 +3464,267 @@ def getModelDefs(self):
                             'ro': True,
                             'doc': 'The x509 certificate sent by the client.'})
                     )),
+                    ('inet:service:platform', {}, (
+
+                        ('url', ('inet:url', {}), {
+                            'ex': 'https://twitter.com',
+                            'doc': 'The primary URL of the platform.'}),
+
+                        ('name', ('str', {'onespace': True, 'lower': True}), {
+                            'ex': 'twitter',
+                            'doc': 'A friendly name for the platform.'}),
+
+                        ('provider', ('ou:org', {}), {
+                            'doc': 'The organization which operates the platform.'}),
+
+                        ('provider:name', ('ou:name', {}), {
+                            'doc': 'The name of the organization which operates the platform.'}),
+                    )),
+
+                    ('inet:service:instance', {}, (
+
+                        ('id', ('str', {'strip': True}), {
+                            'ex': 'B8ZS2',
+                            'doc': 'A platform specific ID to identify the service instance.'}),
+
+                        ('platform', ('inet:service:platform', {}), {
+                            'ex': '{ inet:service:platform:name=slack }',
+                            'doc': 'The platform which defines the service instance.'}),
+
+                        ('url', ('inet:url', {}), {
+                            'ex': 'https://v.vtx.lk/slack',
+                            'doc': 'The primary URL which identifies the service instance.'}),
+
+                        ('name', ('str', {'lower': True, 'onespace': True}), {
+                            'ex': 'synapse users slack',
+                            'doc': 'The name of the service instance.'}),
+                    )),
+
+                    ('inet:service:account', {}, (
```

Comment:
Is the :profile prop intended to capture the public profile information (e.g., invisigoth) ? If so, then do we need a :contact prop to capture the non-public contact information (e.g., visi start, 123 Nowhere St)?